geometry_msgs/PoseStamped tag_pose
int32 tag_id
int32[2] grid_coords